Source/Scene/Cesium3DTileset.js

```diff
@@ -1237,6 +1239,22 @@ define([
             get : function() {
                 return this._ellipsoid;
             }
+        },
+
+        /**
+         * Application specific metadata.
+         *
+         * @memberof Cesium3DTileset.prototype
+         *
+         * @type {Object}
```

Comment:
Change to `@type {*}`, `extras` is not necessarily an object, it could be a string, number, etc.